spec/requests/api/v2/measures_controller_spec.rb

```diff
@@ -12,5 +12,13 @@
     end
 
     it_behaves_like 'a successful jsonapi response'
+
+    it 'the TimeMachine receives the correct Date' do
+      allow(TimeMachine).to receive(:at).and_call_original
+
+      make_request
+
+      expect(TimeMachine).to have_received(:at).with(Time.zone.today)
```

Comment:
This is testing time machine rather then external behaviour and I'm pretty sure would pass even without the `.actual` change to the controller. 

I'd recommend a spec like this instead

```ruby
context 'for historical measure' do
  let(:measure) { create(:measure, validity_end_date: 2.days.ago) }

  it { is_expected.to have_http_status :not_found }
end
```